qiskit/visualization/state_visualization.py

```diff
@@ -272,6 +272,21 @@ def plot_bloch_multivector(
             qc.h(0)
             qc.cx(0, 1)
 
+            state = Statevector.from_instruction(qc)
+            plot_bloch_multivector(state, title="New Bloch Multivector", reverse_bits=False)
+
+    Example - without entanglement:
+        .. jupyter-execute::
+
+            from qiskit import QuantumCircuit
+            from qiskit.quantum_info import Statevector
+            from qiskit.visualization import plot_bloch_multivector
+            %matplotlib inline
+
+            qc = QuantumCircuit(2)
+            qc.h(0)
+            qc.x(1)
+
             state = Statevector.from_instruction(qc)
             plot_bloch_multivector(state, title="New Bloch Multivector", reverse_bits=False)
```

Comment:
I know you were just copying what was above, but it'd be good to get rid of the useless `title` and `reverse_bits` arguments to this function, just to make things a bit clearer.